packages/nvidia_nat_eval/tests/eval/test_atif_adapter.py

```
@@ -0,0 +1,75 @@
+                         expected_output_obj="expected",
+                         output_obj="actual",
+                         trajectory=[step],
+                         full_dataset_entry={"id": item_id})
+
+
+class _CountingConverter:
+
+    def __init__(self) -> None:
+        self.calls = 0
+
+    def convert(self, steps: list[IntermediateStep], *, session_id: str | None = None, agent_name: str | None = None):
```

Ruff (0.15.4)

[warning] Unused method argument: `steps`

(ARG002)